### PR TITLE
Mui finefoods dashboard page

### PR DESCRIPTION
**Pull request by salihozdemir**
Test me! 'MUI'
[Link to DASHBOARD-PAGE](https://dashboa-refine.pankod.com)

Hey, I'm client! Test me! 'MUI'
[Link to DASHBOARD-PAGE](https://dashboa-client.refine.pankod.com)

Test me! 'MUI'
[Link to DASHBOARD-PAGE](https://dashboa-refine.pankod.com)

Hey, I'm client! Test me! 'MUI'
[Link to DASHBOARD-PAGE](https://dashboa-client.refine.pankod.com)

Test me! 'MUI'
[Link to DASHBOARD-PAGE](https://dashboa-refine.pankod.com)

Test me! 'MUI'
[Link to DASHBOARD-PAGE](https://dashboa-refine.pankod.com)

Add Dashboard Page for Mui FineFoods example.

### CODE REVIEW

File: examples/fineFoods/admin/mui/src/components/dashboard/orderTimeline/index.tsx [hunk 1]
```diff
@@ -0,0 +1,152 @@
...
+                    const color = orderStatusColor(status.id.toString())?.color;
+                    const dotColor = orderStatusColor(
+                        status.id.toString(),
+                    )?.dotColor;
+
+                    return (
+                        <TimelineItem key={orderNumber}>
+                            <TimelineOppositeContent
+                                sx={{ display: "none" }}
+                            ></TimelineOppositeContent>
```
Comment (biskuvit) [inline examples/fineFoods/admin/mui/src/components/dashboard/orderTimeline/index.tsx:91]:
```suggestion
                            <TimelineOppositeContent
                                sx={{ display: "none" }}
                           />
```

can it work like this

File: examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx [hunk 2]
```diff
@@ -0,0 +1,196 @@
...
+                                    },
+                                },
+                            });
+                        }}
+                    />,
+                    <GridActionsCellItem
+                        key={2}
+                        icon={<CloseOutlinedIcon color="error" />}
+                        sx={{ padding: "2px 6px" }}
+                        label="Reject"
```
Comment (biskuvit) [inline examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx:147]: we can use translate

**Review by biskuvit — APPROVED** [review]

File: examples/fineFoods/admin/mui/src/components/dashboard/deliveryMap/index.tsx [hunk 3]
```diff
@@ -0,0 +1,71 @@
+import { useList, useNavigation } from "@pankod/refine-core";
+import GoogleMapReact from "google-map-react";
+
+import { MapMarker } from "components/mapMarker";
+import { LocationIcon, CourierIcon } from "components/icons";
+
```
Comment (ozkalai) [inline examples/fineFoods/admin/mui/src/components/dashboard/deliveryMap/index.tsx:6]:
```suggestion

```

File: examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx [hunk 4]
```diff
@@ -0,0 +1,196 @@
...
+    Avatar,
+    Button,
+    DataGrid,
+    GridActionsCellItem,
+    GridColumns,
+    NumberField,
+    Stack,
+    Typography,
+    useDataGrid,
+} from "@pankod/refine-mui";
```
Comment (ozkalai) [inline examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx:16]: global import should locate on the top level

File: examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx [hunk 5]
```diff
@@ -0,0 +1,196 @@
+/* eslint-disable react/display-name */
+import React from "react";
+import { useNavigation, useTranslate, useUpdate } from "@pankod/refine-core";
+
+import { IOrder } from "interfaces";
```
Comment (ozkalai) [inline examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx:5]: local import should be a group after global imports

File: examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx [hunk 6]
```diff
@@ -0,0 +1,196 @@
...
+    GridActionsCellItem,
+    GridColumns,
+    NumberField,
+    Stack,
+    Typography,
+    useDataGrid,
+} from "@pankod/refine-mui";
+
+import CheckOutlinedIcon from "@mui/icons-material/CheckOutlined";
+import CloseOutlinedIcon from "@mui/icons-material/CloseOutlined";
```
Comment (ozkalai) [inline examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx:19]:
```suggestion
import { CheckOutlined,  CloseOutlined }  from "@mui/icons-material;
```

File: examples/fineFoods/admin/mui/src/components/dashboard/orderTimeline/index.tsx [hunk 7]
```diff
@@ -0,0 +1,150 @@
...
+    Tooltip,
+    Typography,
+    Pagination,
+} from "@pankod/refine-mui";
+import dayjs from "dayjs";
+import relativeTime from "dayjs/plugin/relativeTime";
+
+import { IOrder } from "interfaces";
+
+dayjs.extend(relativeTime);
```
Comment (yildirayunlu) [inline examples/fineFoods/admin/mui/src/components/dashboard/orderTimeline/index.tsx:24]: `Dayjs` plugins should be collected in `index.tsx`

Ref: https://github.com/pankod/refine/blob/master/examples/fineFoods/admin/src/index.tsx#L12

**Review by ozkalai — APPROVED** [review]

File: examples/fineFoods/admin/mui/src/components/dashboard/chartTooltip/index.tsx [hunk 8]
```diff
@@ -0,0 +1,30 @@
...
+import { TooltipProps } from "recharts";
+
+type ChartTooltipProps = TooltipProps<number, string> & {
+    suffix?: string;
+};
+
+export const ChartTooltip: React.FC<ChartTooltipProps> = ({
+    active,
+    payload,
+    suffix,
```
Comment (omeraplak) [inline examples/fineFoods/admin/mui/src/components/dashboard/chartTooltip/index.tsx:11]:
```suggestion
    suffix = "",
```

File: examples/fineFoods/admin/mui/src/components/dashboard/chartTooltip/index.tsx [hunk 9]
```diff
@@ -0,0 +1,30 @@
...
+            <Box
+                sx={{
+                    color: "#fff",
+                    fontWeight: 600,
+                    background: "rgba(255, 255, 255, 0.3)",
+                    padding: "4px 8px",
+                    borderRadius: "4px",
+                }}
+            >
+                {payload[0]?.value} {suffix}
```
Comment (omeraplak) [inline examples/fineFoods/admin/mui/src/components/dashboard/chartTooltip/index.tsx:24]:
```suggestion
                {`${payload[0]?.value} ${suffix}`}
```

File: examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx [hunk 10]
```diff
@@ -0,0 +1,196 @@
...
+        () => [
+            {
+                field: "avatar",
+                renderCell: ({ row }) => (
+                    <Avatar
+                        sx={{
+                            width: { xs: 60, sm: 84, lg: 108, xl: 144 },
+                            height: { xs: 60, sm: 84, lg: 108, xl: 144 },
+                        }}
+                        src={row?.products[0]?.images[0].url}
```
Comment (omeraplak) [inline examples/fineFoods/admin/mui/src/components/dashboard/recentOrders/index.tsx:35]:
```suggestion
                        src={row?.products[0]?.images[0]?.url}
```

**Review by omeraplak — APPROVED** [review]
🎖️

**Pull request merged**